controller/tests/models/domain/rsptag_test.py

```diff
@@ -359,3 +387,16 @@ def test_from_str() -> None:
     # them in dictionary form.
     for tag, expected in test_cases.items():
         assert asdict(RSPImageTag.from_str(tag)) == expected
+
+
+def test_age() -> None:
+    """Test that the age() method works with and without supplied basis."""
+    rsptag = RSPImageTag.from_str("d_2021_05_27")
+    now = datetime.now(tz=UTC)
+    # If the next three lines take longer than one second, you have a problem.
+    orig_age = rsptag.age(now)
+    assert orig_age is not None
+    new_age = rsptag.age()
+    assert new_age is not None
+    assert (new_age - orig_age) > timedelta(seconds=0)
+    assert (new_age - orig_age) < timedelta(seconds=1)
```

Comment:
This still isn't testing the semantics of the function, though. If the implementation of `age` was just `return datetime.now(tz=UTC) - datetime.now(tz=UTC)`, it might pass this test.

```python
    assert rsptag.age(datetime(2021, 5, 28, tzinfo=UTC)) == timedelta(days=1)
    now = datetime.now(tz=UTC)
    expected = now - datetime(2021, 5, 27, tzinfo=UTC)
    assert expected <= rsptag.age() <= expected + timedelta(seconds=1)
```
would test the function itself.